### DISCUSSION TITLE
Translating "Display name" in Select Options

### DISCUSSION THREAD

**Discussion by roxithromycinrttgtg** [pimcore/pimcore#16568] [Q&A / Support]
I'm trying to translate my values in a Select Options, but I can't seem to get it to work.
There's no localized field on the rows itself, and my values does not appear in the admin translation as posted here: https://github.com/pimcore/pimcore/issues/3173#issuecomment-409870134

**Comment by NikitaKharkov** [comment]
If I understand correctly the task you should set the option provider.

[Here is the link](https://pimcore.com/docs/platform/Pimcore/Objects/Object_Classes/Data_Types/Dynamic_Select_Types/)

In your provider you will have a **context** param where you will have a parent. And, depends on it you can get the locale and translate the options depends on parent locale.
